specification/securitydevops/resource-manager/Microsoft.SecurityDevOps/preview/2022-09-01-preview/securitydevops.json

```diff
@@ -0,0 +1,2692 @@
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/GitHubRepo"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "GitHubRepo_Get": {
+            "$ref": "./examples/GitHubRepoGet.json"
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "default"
+        ],
+        "summary": "Create or update a monitored GitHub repository.",
+        "operationId": "GitHubRepo_CreateOrUpdate",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/GitHubConnectorName"
+          },
+          {
+            "$ref": "#/parameters/GitHubOwnerName"
+          },
+          {
+            "$ref": "#/parameters/GitHubRepoName"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "in": "body",
+            "name": "gitHubRepo",
+            "description": "Github repo.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/GitHubRepo"
+            }
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Created",
+            "schema": {
+              "$ref": "#/definitions/GitHubRepo"
+            }
+          },
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/GitHubRepo"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "azure-async-operation"
+        },
+        "x-ms-examples": {
+          "GitHubRepo_CreateOrUpdate": {
+            "$ref": "./examples/GitHubRepoCreateOrUpdate.json"
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "default"
+        ],
+        "summary": "Patch a monitored GitHub repository.",
```

Comment:
>Patch [](http://example.com/codeflow?start=20&length=5)

Use the 'update' verb for describing what it does in docs. PATCH is just the HTTP verb.

Reply:
sure, we can tackle this in next API version.

Reply:
It won't require a version change, since its just a description change.

Reply:
agreed, but here is why we will tackle this in next API version.
1. given our product is in code freeze and we generate swagger.
2. Additionally we need 7-10 days to test all swagger changes in Prod
3. We are super close to Microsoft Ignite, it is in less than 7 days.

Happy to take fix in next API version.